waiter/src/waiter/core.clj

```diff
@@ -98,6 +98,7 @@
                      "app-name" :app-name-handler-fn
                      "apps" {"" :service-list-handler-fn
                              ["/" "instances"] :instances-list-handler-fn
+                             ["/" :service-id "/signal/" :signal-type "/" :instance-id] :signal-handler-fn
```

Comment:
Did I have an earlier comment about ordering the instance id and signal type by the signature of the scheduler function?
```suggestion
                             ["/" :service-id "/signal/" :instance-id "/" :signal-type] :signal-handler-fn
```